ethstore/src/account/crypto.rs

```diff
@@ -0,0 +1,183 @@
+// Copyright 2015, 2016, 2017 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+
+// Parity is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+use std::iter::repeat;
+use ethkey::Secret;
+use {json, Error, crypto};
+use crypto::Keccak256;
+use random::Random;
+use smallvec::SmallVec;
+use account::{Cipher, Kdf, Aes128Ctr, Pbkdf2, Prf};
+
+#[derive(Debug, PartialEq, Clone)]
+pub struct Crypto {
```

Comment:
I just take this code && moved to separate file, as it has grown a bit :) Also the whole ethstore crate is undocumented - just thought it is not a good idea to document 1/100 of undocumented code :) Anyway - I've tried to add docs for this struct && others I've added. But I wonder - is there some policy on documentation? I mean - if mod makes something pub - it should be documented? Even if this is some technical mod, unavailable to all others, but its parent?